File: TUGAS 5/Tugas 5.1 .pas
{Nama   : Fauzy Ramadan}
{NPM    : 20.14.1.0051}
{Program: Tugas 5.1 .pas}

program Tugas5_1;
uses crt;

var
n : integer;

function hasil (x : integer) : boolean;
begin
hasil := (n mod 2 = 0);
end;

begin
clrscr;
writeln ('Program Function Bilangan Ganjil Genap');
writeln;
write ('Masukan Nilai : '); readln (n);
writeln;

if hasil(n) then
write ('bilangan ini ialah bilangan genap')
else
write ('bilangan ini adalah bilangan ganjil');

readln;
end.

File: TUGAS 5/TUGAS 5.1.cpp
#include <iostream>
using namespace std;

int n;
int hasil (int a){
	int hasil;
	hasil = n%2 == 0;
}

int main()
{
	cout << "Program function bilangan ganjil genap"<<endl;
	cout << "Masukan Nilai : ";
	cin >> n;
	cout << endl;
	
	if (hasil(n)){
		cout << "Bilangan ini bilangan genap";
	} else {
		cout << "Bilangan ini bilangan ganjil";
	}
}
